projects/views.py

```diff
@@ -148,10 +172,40 @@ def view_project(request, project_id):
                     "An unexpected error occurred while attempting "
                     "to remove this user. Please try again",
                 )
+        elif "del_invite" in request.POST:
+            try:
+                email_code = request.POST["email_code"]
+                mapper.remove_invitation(project_id, email_code)
+                messages.success(
+                    request, 'Invitation for "%s" removed' % email_code
+                )
+            except:
+                logger.exception("Failed to delete invitation")
+                messages.error(
+                    request,
+                    "An unexpected error occurred while attempting "
+                    "to remove this invitation. Please try again",
+                )
+        elif "resend_invite" in request.POST:
+            try:
+                email_code = request.POST["email_code"]
+                mapper.resend_invitation(project_id, email_code, request.user, request.get_host())
```

Comment:
This might need a helper function, but let's not put it in the mapper. It can probably just be a helper function in this file.